lib/init/config-initializer.js

```diff
@@ -226,12 +224,28 @@ function processAnswers(answers) {
         config.extends.push("plugin:@typescript-eslint/recommended");
     }
 
+    // add in library information
+    // The configuration of the framework plugins should be placed at the end of extends.
+    if (answers.framework === "react") {
+        config.plugins.push("react");
+        config.extends.push("plugin:react/recommended");
```

Comment:
After this change, `plugin:react/recommended` would come after `airbnb`, so some rules that `airbnb` explicitly turns off (for example, [`react/jsx-key` because it has too many false positives](https://github.com/airbnb/javascript/blob/7916d6f538028e91129e65fd5633b7d6fa85d000/packages/eslint-config-airbnb/rules/react.js#L94-L95)) would be enabled. Perhaps there's actually no need for `plugin:react/recommended` when `airbnb` is selected?

Reply:
yes, but IMHO, the `recommended` config should always be safe to use - if the rule has too many false positives, it should not be enabled in the `recommended`. :)

Reply:
Yes, it looks unusual that shareable config disables plugin's recommended rules, but I think that generally if the selected shareable config uses a plugin, then the shareable config should have precedence over plugin's configs, i.e. we should just not use plugin's configs.